docs/blog/2022-05-11-webops.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Improved WebOps build time 
+tags: [webops]
+---
+
+# Optimized WebOps build time
+
+We updated the WebOps infrastructure to improve the building time by providing a unified system for building and serving nodejs apps on WebOps.
+
+## What has changed?
+
+### WebOps Unified Build 
+
+WebOps Unified Build is a new unified system for WebOps builds that generates the new '.webops' folder to reduce data and improv the building time. 
```

Comment:
Maybe remove all these things related to `.webops`. These are internal details. Leave the before/after build times.